tabby-settings/src/components/settingsTab.component.pug

```diff
@@ -100,6 +100,15 @@
                             .description(translate) Enable automatic installation of updates when they become available.
                         toggle([(ngModel)]='config.store.enableAutomaticUpdates', (ngModelChange)='saveConfiguration()')
 
+                    .form-line(*ngIf='hostApp.platform !== Platform.Web')
```

Comment:
should exclude Linux as well since it's excluded in the backend

also, the setting should be in the Window settings tab